rules.d/60-persistent-hidraw.rules

```diff
@@ -0,0 +1,15 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="persistent_hidraw_end"
+SUBSYSTEM!="hidraw", GOTO="persistent_hidraw_end"
+
+SUBSYSTEMS=="usb", ENV{ID_BUS}=="", IMPORT{builtin}="usb_id"
+
+# by-id links
+KERNEL=="hidraw*", ENV{ID_BUS}=="?*", ATTRS{bInterfaceNumber}=="|00", SYMLINK+="input/by-id/$env{ID_BUS}-$env{ID_SERIAL}-hidraw"
+KERNEL=="hidraw*", ENV{ID_BUS}=="?*", ATTRS{bInterfaceNumber}=="?*", ATTRS{bInterfaceNumber}!="00", SYMLINK+="input/by-id/$env{ID_BUS}-$env{ID_SERIAL}-if$attr{bInterfaceNumber}-hidraw"
+
+# add a more readable 'fido' link for devices with ID_FIDO_TOKEN==1
+KERNEL=="hidraw*", ENV{ID_BUS}=="?*", ENV{ID_FIDO_TOKEN}=="?*", SYMLINK+="input/by-id/$env{ID_BUS}-$env{ID_SERIAL}-fido-hidraw"
+
```

Comment:
i think by-path might make sense too. maybe not so much for fido2 stuff, but for other hidraw stuff it might, you never know. i.e. just do what persistent input devices do to for their by-path symlinks

Reply:
Added.